feat: update 100

brianp opened the pull request:
Pushing up the second draft of the 0.45.0 and StageNet announcement. Bringing to attention our 8-week release cycles, and stabilized test net.

brianp commented:
@CjS77 please take a look over. I'll make another pass of it in the morning, and will make any updates/corrections/changes as per review.

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,68 @@
+
+## Tari Protocol 0.45.0 Released: Your Wait for Mainnet Continues
+
+Congratulations folks, it's that time again! The Tari protocol team is excited to announce the release of version 0.45.0, bringing a number of new features and improvements to the network. This release also marks an important milestone in our development with big news - wait for it - we've launched the Tari StageNet! Yes, you heard it right. We know, you're excited about the mainnet, but trust me ...
+
+### What's so special about StageNet?
+What's so special about StageNet, you ask? Well, after several iterations of testnets with cool names like our currently running Esme, Igor, and our previously retired Dibbler, Weatherwax, Ridcully, and Stibbons (RIP ✌️), we're approaching a time that requires a stable testnet that mirrors our future mainnet as closely as possible. And that's what StageNet is all about. It's a long-lived, stable  ...
+
+I know what you're thinking. Without Mainnet, what's the difference between this stable testnet and the previous ones? Good question! And here's the answer: It's all in how we're caring for it. We're transitioning to an 8-week release cycle schedule! Yes, you heard it right. We may not have a mainnet yet, but we're gonna start developing like we do. So, StageNet will get new updates every 8 weeks ...
+
```
No issues with the light, conversational style, but dial it back 2 notches.

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,68 @@
+
+### How this will look in the immediate future:
+
+Currently, the Protocol team is a highly functioning group, throwing out new features and fixes whenever inspiration strikes. When work gets completed it gets merged into `development`. Then, once every so often we decide "hey we've done some really great work here, we should share it with people". And somebody will tag a release. It's gotten us to where we are but can be unpredictable. So we're  ...
```
```suggestion
Currently, the Protocol team is a highly functioning group, throwing out new features and fixes whenever inspiration strikes. When work gets completed it gets merged into the `development` branch on our [Github repo](link). 

Then, every so often we decide "hey we've done some really great work here, we should share it with people". And somebody will tag a release. This approach has gotten us to where we are but is a little unpredictable. So we're introducing a little more structure to our process.
```

brianp replied:
Updated with suggestion.

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,68 @@
+
+Currently, the Protocol team is a highly functioning group, throwing out new features and fixes whenever inspiration strikes. When work gets completed it gets merged into `development`. Then, once every so often we decide "hey we've done some really great work here, we should share it with people". And somebody will tag a release. It's gotten us to where we are but can be unpredictable. So we're  ...
+
+Going forward, we'll be taking a more meticulous approach to code selection and testing. We'll be running features through the gauntlet of our named testnets to ensure they're truly ready for prime time. No half-baked ideas making their way through, only once a feature has passed the rigorous testing, will it earn its place in the coveted stagenet branch. These features will be pruned and picked  ...
```
Consistent naming

```suggestion
Going forward, we'll be taking a more meticulous approach to code selection and testing. We'll be running features through the gauntlet of our named testnets to ensure they're truly ready for prime time. No half-baked ideas making their way through, only once a feature has passed the rigorous testing, will it earn its place in the coveted StageNet branch. These features will be pruned and picked from our `developmnet` branch into the `stagenet` branch. Only then will a tag be created from this branch, and released for your ultimate enjoyment.
```

brianp replied:
Updated the naming inconsistencies. Also did a scan for some other inconsistent usages "testnets" vs "test networks" and "Mainnet" vs "mainnet"

CjS77 reviewed (commented):
Thanks. Good start. 

* I don't think you link to the contribution guidelines / RFC that describes the process in detail
* Some more colour on the purpose the other testnets will serve would be helpful.
* Any reason you didn't mention feature gates?

CjS77 commented:
I see this is update # 💯 ! 🎉

brianp commented:
>    I don't think you link to the contribution guidelines / RFC that describes the process in detail

It was actually there, it was just _under_ the release notes. So i've moved it up and into the main body.

>    Some more colour on the purpose the other testnets will serve would be helpful.

Added some additional colour.

>    Any reason you didn't mention feature gates?

Yes, there was a reason. But that's been discussed elsewhere now, and I've updated to mention the practicality and use of the feature codes.

I think I'm happy with it in its current form. Let me know if you'd like to see any other additions.

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,74 @@
+class: subpage
+---
+
+## Tari Protocol 0.45.0 Released: Your Wait for Mainnet Continues
```
🤔 
```suggestion
## Tari Protocol 0.45.0 Released: Approaching mainnet one minor release at a time
```

brianp replied:
😈

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,74 @@
+### What's so special about StageNet?
+After several iterations of testnets with cool names like our currently running Esme, Igor, and our previously retired Dibbler, Weatherwax, Ridcully, and Stibbons (RIP ✌️), we're approaching a time that requires a stable testnet that mirrors our future Mainnet as closely as possible. And that's what StageNet is all about. It's a long-lived, stable testnet that won't reset often, because we know h ...
+
+We won't be getting rid of our named testnets though! They still play an integral part in the development of the Tari network. This is where features are tested a little more fast and lose. We'll do our best not to, but these networks may see breakage sometimes. Or a new one may crop up to test a specific subset of features. For example right now our two running named testnets exist for different ...
```
```suggestion
We won't be getting rid of our named testnets though. They still play an integral part in the development of the Tari network, and are where we can play a little more fast and loose with features. We'll do our best not to, but these networks may see breakage sometimes. Or a new one may crop up to test a specific subset of features. For example, right now our two running named testnets exist for different reasons. 

Our `Esme` network has been our stable-er testnet prior to `StageNet`. It sees new features and enhancements as they come out, and supports our mobile wallets [Tari Aurora](https://aurora.tari.com/) on [iOS](https://apps.apple.com/us/app/tari-aurora/id1503654828) and [Android](https://play.google.com/store/apps/details?id=com.tari.android.wallet). 

Our `Igor` testnet is used in the development of the [DAN Layer](https://www.tari.com/updates/2023-02-01-update-99.html) and sees frequent resets to help support the rapid pace of the development team.
```

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,74 @@
+
+We won't be getting rid of our named testnets though! They still play an integral part in the development of the Tari network. This is where features are tested a little more fast and lose. We'll do our best not to, but these networks may see breakage sometimes. Or a new one may crop up to test a specific subset of features. For example right now our two running named testnets exist for different ...
+
+I know what you're thinking. Without Mainnet, what's the difference between this new stable StageNet and the previous ones? And here's the answer: It's all in how we're shepherding it. We're transitioning to an 8-week release cycle schedule! Yes, you heard it right. We may not have a Mainnet yet, but we're gonna start developing like we do. StageNet will get new updates every 8 weeks, just like t ...
```
```suggestion
I know what you're thinking. Without Mainnet, what's the difference between this new stable StageNet and the previous ones? And here's the answer: It's all in how we're shepherding it. We're transitioning to an 8-week release cycle schedule. Yes, you heard it right. We may not have a Mainnet yet, but we're gonna start developing like we do. StageNet will get new updates every 8 weeks, just like the Mainnet will soon™️. This is a big change for our protocol team's workflow, and we're excited to iron out the kinks early to ensure a smoother Mainnet transition.
```

CjS77 commented on _updates/2023-02-17-update-100.md:
```diff
@@ -0,0 +1,74 @@
+
+Then, every so often we decide "hey we've done some really great work here, we should share it with people". And somebody will tag a release. This approach has gotten us to where we are but is a little unpredictable. So we're introducing a little more structure to our process.
+
+Going forward, we'll be taking a more meticulous approach to code selection and testing. We'll be running features through the gauntlet of our named testnets to ensure they're truly ready for prime time. No half-baked ideas making their way through, only once a feature has passed the rigorous testing, will it earn its place in the coveted StageNet branch. These features will be pruned and picked  ...
```
```suggestion
Going forward, we'll be taking a more meticulous approach to code selection and testing. We'll be running features through the gauntlet of our named testnets to ensure they're truly ready for prime time. No half-baked ideas muddling their way in, only once a feature has passed the rigorous testing will it earn its place in the coveted StageNet branch. These features will be pruned and picked from our `development` branch into the `stagenet` branch. Only then will a tag be created from this branch, and released for your ultimate enjoyment.
```

CjS77 reviewed (commented):
Content ACK. Made several editorial suggestions.

brianp commented:
Reviewed the suggestions and all changes looked good to me.

CjS77 reviewed (approved):
LGTM
